evaluator/src/test/kotlin/LicenseViewTest.kt

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.evaluator
+
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.WordSpec
+
+class LicenseViewTest : WordSpec() {
+    init {
+        "All" should {
+            "return the correct licenses" {
+                LicenseView.All.licenses(packageWithoutLicense, emptyList()) shouldBe emptyList()
+
+                LicenseView.All.licenses(packageWithoutLicense, detectedLicenses) shouldBe listOf(
+                    Pair("LicenseRef-a", LicenseSource.DETECTED),
+                    Pair("LicenseRef-b", LicenseSource.DETECTED)
+                )
+
+                LicenseView.All.licenses(packageWithOnlyConcludedLicense, emptyList()) shouldBe listOf(
```

Comment:
This API kind of suggests that it abstracts from the license source - but on the other hand it spits out tuples containing the license source ?  Is this actually needed - because without that I'd imagine the tests to read much nicer.

You could then use license IDs like: "concluded-license-a"